webclient/app/src/app/pages/appEditor/HandleRelations.jsx

```diff
@@ -0,0 +1,30 @@
+import {SteppedLineTo} from "react-lineto";
+
+function updateRelationCoordinates(entities) {
+    let coordinates = [];
+    entities.forEach(entity => {
+        if (entity.relationships) {
+            entity?.relationships.forEach(relationship => {
+                coordinates.push({
+                    from: 'anchor_' + entity.name, to: 'anchor_' + relationship.otherEntityName
+                })
+            })
+        }
+    })
+    return coordinates;
+}
+
+function renderRelations(coordinates, theme) {
```

Comment:
Why did you export this to an extra file? This is just used inside the ERSection